src/ballet/aes/fd_aes_gcm.h

```diff
@@ -78,33 +134,36 @@ fd_aes_256_gcm_init( fd_aes_gcm_t * aes_gcm,
    (AAD serves to mix in arbitrary additional data into the auth tag,
    such that tampering with the AAD results in a decryption failure)
 
-   fd_aes_gcm_aead_encrypt reads plaintext from p, writes ciphertext to
+   fd_aes_gcm_encrypt reads plaintext from p, writes ciphertext to
    c, and writes the auth tag to 'tag'.  encrypt cannot fail.
 
-   fd_aes_gcm_aead_decrypt reads the expected auth tag and ciphertext,
+   fd_aes_gcm_decrypt reads the expected auth tag and ciphertext,
    and writes the decrypted plaintext to p.  Ciphertext and auth tag are
    usually transmitted as-is over a network packet.  Returns 1 on
    success, or 0 on failure.  Reasons for failure include:  Corrupt
    ciphertext, corrupt sz, corrupt AAD, or corrupt tag (could be due to
    network corruption or malicious tampering). */
 
 void
-fd_aes_gcm_aead_encrypt( fd_aes_gcm_t * aes_gcm, /* rdi */
-                         uchar *        c,       /* rsi */
-                         uchar const *  p,       /* rdx */
-                         ulong          sz,      /* rcx */
-                         uchar const *  aad,     /* r8  */
-                         ulong          aad_sz,  /* r9  */
-                         uchar          tag[ static 16 ] );
+fd_aes_gcm_encrypt( fd_aes_gcm_t * aes_gcm,
+                    uchar *        c,
+                    uchar const *  p,
+                    ulong          sz,
+                    uchar const *  aad,
+                    ulong          aad_sz,
+                    uchar          tag[ static 16 ] );
 
 int
-fd_aes_gcm_aead_decrypt( fd_aes_gcm_t * aes_gcm,
-                         uchar const *  c,
-                         uchar *        p,
-                         ulong          sz,
-                         uchar const *  aad,
-                         ulong          aad_sz,
-                         uchar const    tag[ static 16 ] );
+fd_aes_gcm_decrypt( fd_aes_gcm_t * aes_gcm,
+                    uchar const *  c,
+                    uchar *        p,
+                    ulong          sz,
+                    uchar const *  aad,
+                    ulong          aad_sz,
+                    uchar const    tag[ static 16 ] );
+
+#define FD_AES_GCM_DECRYPT_FAIL (0)
+#define FD_AES_GCM_DECRYPT_OK   (1)
```

Comment:
nit: they may go up, before FD_PROTOTYPE_START

Reply:
I'd rather keep them close to the function they relate to